challenges/functions.js

```diff
@@ -6,25 +6,43 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+ function consume(param1, param2, cb) {
+   console.log(cb(param1, param2))
+}
+
+consume('burgers', 'pizza', function(param1, param2) {
+   console.log(param1, param2);
```

Comment:
This is returning an undefined because you are console.logging instead of returning anything.